### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by sdepold**
This PR adds some stuff that is on the current website. As I’m fine with actually completely migrating to rtd I’d like to prepare the landing page and probably something else first :) Will provide some other PRs later.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-site_name: Sequelize - node.js ORM for Postgres, MySQl, MariaDB and SQLite
+site_name: Sequelize | The Node.js ORM
```
Comment (mickhansen) [inline mkdocs.yml]: The dialects might be valuable for SEO, we also might want to add io.js

Comment (sdepold) [inline mkdocs.yml]: Agreed. Is there a way to not have the site_name in the sidebar?

File: docs/readthedocs_theme/base.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   {% block htmltitle %}
   <title>{{ page_title }}</title>
+  {% if page_description %}<meta name="description" content="{{ page_description }}">{% endif %}
```
Comment (sdepold) [inline docs/readthedocs_theme/base.html]: Will this change be available on rtd as well?

Comment (janmeier) [inline docs/readthedocs_theme/base.html]: Not sure .. lets find out http://sequelize.readthedocs.org/en/feature-docs/ :)

File: mkdocs.yml (hunk 1 above)
Comment (mickhansen) [inline mkdocs.yml]: Dnu, @janmeier knows this stuff, not me :D

File: docs/readthedocs_theme/base.html (hunk 2 above)
Comment (sdepold) [inline docs/readthedocs_theme/base.html]: Nice :) Alright seems to work. Good to know. So the page title itself is shortened now but the page description contains the keywords. Is that OK ? We can also just overwrite the wording in the left-hand navigation block.

Comment (sdepold) [inline docs/readthedocs_theme/base.html]: Btw. I tried this yesterday: http://docs.sequelizejs.com

Comment (sdepold) [inline docs/readthedocs_theme/base.html]: Just checked the currently available schema with my wife (who is doing SEO stuff) and it seems that the dialects are mostly pointless anyways as the titles are too long for google. 

Example:

```Working with legacy tables - Sequelize - node.js ORM for Postgres, MySQl, MariaDB and SQLite```

-->

Peeps will see / Google will rate only:

serp

Having that said I'd vote for the shorter title.

**Comment by Sarii87** [comment]
That's me!

File: docs/readthedocs_theme/base.html (hunk 2 above)
Comment (Sarii87) [inline docs/readthedocs_theme/base.html]: That's me!

File: mkdocs.yml [hunk 3]
```diff
@@ -1,4 +1,6 @@
...
+site_description: Sequelize is a promise-based ORM for Node.js and io.js. It supports the dialects PostgreSQL, MySQL, MariaDB, SQLite and MSSQL.
```
Comment (fixe) [inline mkdocs.yml]: I would remove "promise-based" from the description.

File: docs/index.md [hunk 4]
```diff
@@ -16,8 +23,8 @@ return sequelize.sync().then(function() {
   return User.create({
     username: 'sdepold',
     birthday: new Date(1986, 06, 28)
-  }).then(function(sdepold) {
-    console.log(sdepold.values)
   });
+}).then(function(sdepold) {
```
Comment (fixe) [inline docs/index.md]: What about renaming `sdepold` to `foobar`?

Comment (mickhansen) [inline docs/index.md]: Yeah `user` would be better.

File: docs/readthedocs_theme/base.html (hunk 2 above)
Comment (mickhansen) [inline docs/readthedocs_theme/base.html]: @sdepold that doesnt matter for the frontpage tho.

File: mkdocs.yml (hunk 3 above)
Comment (mickhansen) [inline mkdocs.yml]: Agreed, it's leftover from when we changed it from event based i guess heh.

File: docs/readthedocs_theme/base.html (hunk 2 above)
Comment (mickhansen) [inline docs/readthedocs_theme/base.html]: Ideally we would have a title for the frontpage and a different title for subpages which have their one title.

File: mkdocs.yml (hunk 1 above)
Comment (mickhansen) [inline mkdocs.yml]: @sdepold ORM alone is too generic, we are SQL focused, that matters a great deal. Hence why it also matters that people might search for `Node postgres orm`

File: docs/readthedocs_theme/base.html (hunk 2 above)
Comment (mickhansen) [inline docs/readthedocs_theme/base.html]: Although obviously, sequelizejs.com should be the primary source of SEO stuff, but might as well have the basics on the docs so they are easy to find :)

**Comment by sdepold** [comment]
:+1: Will update the PR tomorrow!

**Comment by sdepold** [comment]
I think I addressed all of the comments

**Pull request merged**